src/mkpsxiso/iso.cpp

```diff
@@ -801,59 +801,58 @@ void iso::DirTreeClass::OutputLBAlisting(FILE* fp, int level) const
 		else if ( entry.type == EntryType::EntryFile )
 		{
 			fprintf( fp, "File  " );
-			fprintf( fp, "%-17s", entry.id.c_str() );
+			fprintf( fp, "%-17s", CleanIdentifier(entry.id).c_str() );
 			fprintf( fp, "%-10" PRIu32, GetSizeInSectors(entry.length) );
 		}
 		else if ( entry.type == EntryType::EntryDir )
 		{
 			fprintf( fp, "Dir   " );
-			fprintf( fp, "%-17s", entry.id.c_str() );
+			fprintf( fp, "%-17s", CleanIdentifier(entry.id).c_str() );
 			fprintf( fp, "%-10s", "" );
 		}
 		else if ( entry.type == EntryType::EntryXA )
 		{
 			fprintf( fp, "XA    " );
-			fprintf( fp, "%-17s", entry.id.c_str() );
+			fprintf( fp, "%-17s", CleanIdentifier(entry.id).c_str() );
 			fprintf( fp, "%-10" PRIu32, GetSizeInSectors(entry.length, 2336) );
 		}
 		else if ( entry.type == EntryType::EntryXA_DO )
 		{
 			fprintf( fp, "XA    " );
-			fprintf( fp, "%-17s", entry.id.c_str() );
+			fprintf( fp, "%-17s", CleanIdentifier(entry.id).c_str() );
 			fprintf( fp, "%-10" PRIu32, GetSizeInSectors(entry.length) );
 		}
 		else if ( entry.type == EntryType::EntryDA )
 		{
 			fprintf( fp, "CDDA  " );
-			fprintf( fp, "%-17s", entry.id.c_str() );
+			fprintf( fp, "%-17s", CleanIdentifier(entry.id).c_str() );
 			fprintf( fp, "%-10" PRIu32, 150 + GetSizeInSectors(entry.length, CD_SECTOR_SIZE) );
 		}
 
 		// Write LBA offset
 		fprintf( fp, "%-10d", entry.lba );
 
 		// Write Timecode
-		fprintf( fp, "%-12s", SectorsToTimecode(150 + entry.lba).c_str());
-
-		// Write size in byte units
 		if (entry.type != EntryType::EntryDir)
 		{
-			fprintf( fp, "%-10" PRId64, entry.length );
-		}
-		else
-		{
-			fprintf( fp, "%-10s", "" );
-		}
+			fprintf( fp, "%-12s", SectorsToTimecode(150 + entry.lba).c_str());

```

Comment:
As DA files are not present in the DATA track, thus not having them an BCD time in the sector header address, should I keep calculating its real time for the -lba log or a time to match the ones in the .cue file?